CHANGELOG.md

```diff
@@ -4,7 +4,13 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
-## Unreleased
+## [5.7.0] - 2022-02-08
+### Added
+- Support for signed integers
+
+## [5.6.1] - 2022-02-07
```

Comment:
This release(5.6.1) is not in development yet. 
I think it should be updated when rebased with develop.

Reply:
-removed these releases from the changelog
-once this feature is in a release i can update changelog

Reply:
Remove the version, since it is wasn't released yet.
Remove 
- [5.7.0] - 2022-02-08
- [5.6.1] - 2022-02-07

Keep only as Unreleased 
```
## Unreleased
### Added
- Support for signed integers

### Changed
- Updated RPCSelector Timestamp strategy to not take too much time to run
```

Reply:
changed